htsinfer/exceptions.py

```diff
@@ -44,3 +44,9 @@ class TranscriptsFastaProblem(Exception):
 
 class CutadaptProblem(Exception):
     """Exception raised when running cutadapt commands."""
+
+
+class UnsupportedSampleSourceException(Exception):
+    """Exception raised when taxonomy ID is not found in the source
+    organism list.
```

Comment:
According to [PEP 8](https://peps.python.org/pep-0008/) and [PEP 257](https://peps.python.org/pep-0257/), docstrings should either fit on one line:

```py
"""One-line docstring."""
```

OR they should be broken apart into a summary line and an extended description (also, for any multiline docstring, the closing `"""` should be on their own line):

```python
"""A multiline docstring.

With an extended description.
"""
```

In this case,  I would just keep it on one line by just saying:
| [...] taxonomy ID is not supported.

Reply:
Thanks for the review, I refactored it :)